usage-rules.md

```diff
@@ -521,7 +520,7 @@ Relationships describe connections between resources and are a core component of
 
 #### Relationship Types
 
-- For Polymorphic relationships, you can model them using `Ash.Type.Union`; see the “Polymorphic Relationships” guide for more information.
+- For Polymorphic relationships, you can model them using `Ash.Type.Union`; see the [Polymorphic Relationships guide](https://hexdocs.pm/ash/polymorphic-relationships.html) for more information.
```

Comment:
We intentionally left out the link here because we want it to search in the current version of the docs that it has using `mix usage_rules.search_docs`, not fetch the latest version.